packages/ontario-design-system-component-library-react/src/react-component-lib/index.ts

```diff
@@ -1,2 +1,2 @@
-export { createReactComponent } from './createComponent';
-export { createOverlayComponent } from './createOverlayComponent';
+export { createReactComponent } from './createComponent.js';
+export { createOverlayComponent } from './createOverlayComponent.js';
```

Comment:
So these were not being found without the suffix because es6 modules need the extension to run in the browser. However, this was fixed by an earlier change to moduleResolution so this can be changed now. This overlay stuff was pushed out of my mind after the pains it caused I think.